schema/cassandra/cadence/schema.cql

```diff
@@ -260,13 +260,21 @@ CREATE TYPE task (
   partition_config map<text, text>
 );
 
+CREATE TYPE task_list_partition_config (
```

Comment:
@natemort can you check this type and give feedback on what fields you would need for upcoming zonal isolation improvements? 
It's better to avoid extra schema changes but if we don't know the shape of it then it's fine. We can come back and change this later.

Reply:
Discussed with Zijian, this approach is fine and isn't incompatible. We'll make additional changes in the future since things are a little up in the air still.